From: Steve Magoun
Subject: irc: periodic '/who' polling causes connection drops
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/pidgin/+bug/856631

Index: pidgin-2.10.2/libpurple/protocols/irc/msgs.c
===================================================================
--- pidgin-2.10.2.orig/libpurple/protocols/irc/msgs.c	2012-03-14 06:28:00.000000000 +0100
+++ pidgin-2.10.2/libpurple/protocols/irc/msgs.c	2012-03-16 12:34:04.784066122 +0100
@@ -110,8 +110,10 @@
 	irc_blist_timeout(irc);
 	if (!irc->timer)
 		irc->timer = purple_timeout_add_seconds(45, (GSourceFunc)irc_blist_timeout, (gpointer)irc);
+    /*
     if (!irc->who_channel_timer)
         irc->who_channel_timer = purple_timeout_add_seconds(300, (GSourceFunc)irc_who_channel_timeout, (gpointer)irc);
+    */
 }
 
 void irc_msg_default(struct irc_conn *irc, const char *name, const char *from, char **args)
